db/migrate/data/20190617191109_move_awx_credentials_to_authentications/standalone_decrypt.py

```diff
@@ -0,0 +1,58 @@
+import base64
```

Comment:
I think we should somehow reference the source of this code, in case we have to follow it back later.

Reply:
I did reference it in the commit message https://github.com/ManageIQ/manageiq-schema/pull/385/commits/89787bad5841b54d571221aa39c4eac15cd531bf but I can definitely put something in the source as well.

Do you have a format in mind?

Reply:
A few questions to piggy back off of this:

Maybe github or some other direct links to source above each method you are ripping off here? (I am under the assumption that is what you did based on your commit message, without looking at the source yet)

Can we put the assumed `django` and `ansible` versions in here as reference?  While I expect these libs to be pretty static, it would be helpful in the scenario where a customer may have upgraded `ansible` on the appliance, and now we have different versions to contend with.

Reply:
The version of ansible isn't really relevant as this is and ansible-tower only detail and we control the version of tower.

I'm installing django `2.2.2` in https://github.com/ManageIQ/manageiq-appliance-build/pull/324, but can reference that here as well.

Reply:
@carbonin for what it is worth, it seems that `awx` uses `django-1.11.1` on the reference appliance I am using.

> The version of ansible isn't really relevant


Yup, derp on my part.

Reply:
I'm seeing version `1.11.16` on my appliance, but yeah, good point. Let me change that over there.